plugins/commands/compare/src/main/kotlin/CompareCommand.kt

```diff
@@ -93,6 +94,11 @@ class CompareCommand(descriptor: PluginDescriptor = CompareCommandFactory.descri
         help = "Ignore environment differences."
     ).flag()
 
+    private val ignoreToolVersions by option(
+        "--ignore-tool-versions", "-v",
```

Comment:
The short `-v` is commonly used as `--verbose`. 
Should we simply drop the short version here to avoid confusion?

Reply:
ok, then maybe let's keep it for now.